types/dec_coin_test.go

```diff
@@ -1125,7 +1125,7 @@ func (s *decCoinTestSuite) TestDecCoin_ParseDecCoin() {
 		// throw error when upper case characters are used. Currently uppercase denoms are allowed.
 		{"Invalid denom", "9.3STAKE", sdk.DecCoin{"STAKE", sdk.NewDecWithPrec(93, 1)}, false},
```

Comment:
I think this test case name should be `valid upper case denom` and then remove the TODO